sql/src/test/resources/model/model_file_csv.json

```diff
@@ -0,0 +1,27 @@
+{
+  version: '1.0',
```

Comment:
Calcite uses Jackson in a lenient mode (e.g. it allows unquoted attribute names and comments). But you are welcome to use strict JSON.